lblod/frontend-lokaal-mandatenbeheer
# add possibility for admin to impersonate users

## Description

add possibility for admin to impersonate users

Note: the icons are slightly different in loket, this is because they use a more recent appuniversum version. Updating seems to require bumping ember major version, which merits its own story.
## How to test

Log in as Agentschap Binnenlands Bestuur, click the impersonate icon in the top bar:

![image](https://github.com/user-attachments/assets/8833b641-7359-470b-9e12-e1f857d3faa3)

![image](https://github.com/user-attachments/assets/d8974926-10d2-4a05-a1a2-3bcf9ea5850d)

Select another user:
![image](https://github.com/user-attachments/assets/f0e8b567-ccee-451e-9862-5b6eacc591ff)

See that you can see what this user sees

![image](https://github.com/user-attachments/assets/4af8a126-7029-43c9-b86e-0f21f5d1ba41)

End the impersonation session:
![image](https://github.com/user-attachments/assets/3c1a176d-e23f-4ecf-884c-c247775c5098)

## Links to other PR's
Requires these changes on the app side to work:
- https://github.com/lblod/app-lokaal-mandatenbeheer/pull/211
